setup.cfg

```diff
@@ -12,6 +12,12 @@ platforms = Linux, Mac OS X, Windows
 keywords = ipython, jupyter
 classifiers =
     Development Status :: 5 - Production/Stable
+    Framework :: Jupyter
+    Framework :: Jupyter :: JupyterLab
+    Framework :: Jupyter :: JupyterLab :: 4
+    Framework :: Jupyter :: JupyterLab :: Extensions
+    Framework :: Jupyter :: JupyterLab :: Extensions :: Themes
+    Framework :: Jupyter :: JupyterLab :: Extensions :: Media Types
```

Comment:
Maybe better to skip this as we would want to avoid JupyterLab itself showing up on the list of _extensions_?